code_samples/api/public_php_api/src/Command/ViewContentMetaDataCommand.php

```diff
@@ -99,10 +99,11 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         // Relations
         $versionInfo = $this->contentService->loadVersionInfo($contentInfo);
-        $relations = $this->contentService->loadRelations($versionInfo);
-        foreach ($relations as $relation) {
-            $name = $relation->destinationContentInfo->name;
-            $output->writeln('Relation to content ' . $name);
+        $relationCount = $this->contentService->countRelations($versionInfo);
+        $relationList = $this->contentService->loadRelationList($versionInfo, 0, $relationCount);
+        foreach ($relationList as $relationListItem) {
+            $name = $relationListItem->getRelation()->destinationContentInfo->name;
```

Comment:
not sure how we care about that in doc samples, but technicaly `getRelation` could return null, so either we need `$relationListItem->getRelation()?->destinationContentInfo->name` or `hasRelation` check before that.

Reply:
Yup it should, thats why I knew about it. Not sure why it wasnt catched here.

Reply:
```suggestion
            $name = $relationListItem->hasRelation() ? $relationListItem->getRelation()->destinationContentInfo->name : '(N/A)';
```